src/uiprotect/data/base.py

```diff
@@ -495,6 +495,7 @@ def update_from_dict(cls: ProtectObject, data: dict[str, Any]) -> ProtectObject:
         value: Any
 
         for key, item in data.items():
+            key = to_snake_case(key)
```

Comment:
This is likely to be rather expensive to run on everything.

Reply:
maybe something like `obj_dict  = {to_snake_case(k): v for k, v in obj_dict.items()}`